app/src/main/java/org/mydaily/ui/adapter/ReportDetailAdapter.kt

```diff
@@ -0,0 +1,39 @@
+package org.mydaily.ui.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import org.mydaily.R
+import org.mydaily.data.model.ReportListData
+import org.mydaily.databinding.ReportListItemBinding
+
+class ReportDetailAdapter(private val context: Context) : RecyclerView.Adapter<ReportDetailAdapter.ViewHolder>() {
```

Comment:
생성자로 context는 왜 전달해줘야하나요?

Reply:
```suggestion
class ReportDetailAdapter(private val context: Context) : RecyclerView.Adapter<ReportDetailAdapter.ViewHolder>() {
...
binding = DataBindingUtil.inflate(
            LayoutInflater.from(context),
            R.layout.report_list_item, parent, false)
        return ViewHolder(binding)
```

확인해본 결과 ViewHolder에 바인딩 을 넘겨줄때 LayoutInflater.from(context)에서 사용하기 위해 전달해주었습니다.

Reply:
그부분은 `parent.context` 로도 가능합니다!